Revamp readme

stelios-ritual opened the pull request:
## Motivation

- Revame README
- Update versions
- Anvil works out of the box

evangelos-ritual reviewed (commented):
Great!

Also on the main branch, container id names should adhere to `[a-zA-Z0-9][a-zA-Z0-9_.-]+`

evangelos-ritual reviewed (commented):
- After adding a service,

In node version 1.2.0, `"port": 3000` is missing

In node version 1.3.0 the empty config params produce error:
`[error    ] Config validation error: field 'accepted_payments'. Input should be a valid dictionary [shared.container] `

- When trying to specify version `infernet-cli config anvil -v "1.3.0"` it allows only for 1.3.0 but the node image is always 1.2.0 in docker compose

stelios-ritual commented:
Please note, recipes / CLI are only supposed to be compatible with node v1.3.0+, for that exact reason (auto-assigning ports etc.)

> In node version 1.3.0 the empty config params produce error:
[error    ] Config validation error: field 'accepted_payments'. Input should be a valid dictionary [shared.container] 
fixed

> When trying to specify version infernet-cli config anvil -v "1.3.0" it allows only for 1.3.0 but the node image is always 1.2.0 in docker compose
fixed

@evangelos-ritual

evangelos-ritual commented on services/hf-client-inference/1.0.0/recipe.json:
```diff
@@ -1,10 +1,18 @@
     "config": {
-        "id": "hf-client-inference:1.0.0",
+        "id": "hf-client-inference-1.0.0",
         "image": "ritualnetwork/hf_client_inference_service:1.0.0",
```
Monorepo name and image name is `hf_inference_client_service`

stelios-ritual replied:
good catch

evangelos-ritual reviewed (commented):
Great, ran it across all networks, + added css_service and hf_service, works fine!
